h2o-docs/src/product/faq/clusters.rst

```diff
@@ -234,3 +234,13 @@ The following information displays for each message:
    retrieves the ID of a previously unseen type
 -  ``bytes``: Information extracted from the message, including the type
    of the task and the unique task number
+
+--------------
+
+Why do I receive GC (Allocation Failure)?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The G1 collector copies live data out of one region (evacuation) and into another. You will receive a "[GC (Allocation Failure)]" when there is no more space left to allocate live objects from the region being evacuated.
```

Comment:
Add a little more info:

"... being evacuated. Note that this is a normal part of GC operations, and the occurrence of these messages doesn't directly indicate an issue."